### PR TITLE
#7 fixed uninitilized sysPropertyLogLevel

### LINKED ISSUE #7: [BUG] LogLevel of Default Logger should never be null
**Describe the bug**

If the system property for the DefaultLogger in the API module is not set, then the `sysPropertyLogLevel` value is never set. This leads to not setting the `logLevel` of a new DefaultLogger instance, causing a NullPointerException when calling log methods.

**To Reproduce**
1. Only use the api dependency.
2. Run any application using the DefaultLogger without setting the systemLogLevel
3. Call any logging method of the Logger.

**Expected behavior**
The Logger should be initialized with a default LogLevel

**OPTIONAL Priority**
Major

### PR DESCRIPTION
**Pull request by pointan**
Closes #7 

Updated the sysPropertyLogLevel in DefaultLogger of the API module to be
default initialized with WARN instead of null, which caused
NullPointerException if no LogLevel was set fia the system property

### CODE REVIEW

File: api/src/main/java/science/aist/seshat/DefaultLogger.java [hunk 1]
```diff
@@ -27,14 +27,15 @@
...
-    private static LogLevel sysPropertyLogLevel = null;
+    private static LogLevel sysPropertyLogLevel;
 
     static {
         try {
             String systemLogLevel = System.getProperty(LOG_LEVEL_SYSTEM_PROPERTY_NAME);
             sysPropertyLogLevel = science.aist.seshat.LogLevel.valueOf(systemLogLevel);
         } catch (Exception e) { // Just catch any exception if the log level could not have been extracted
             System.err.println("Could not load log level from system property: " + e.getMessage());
+            sysPropertyLogLevel = LogLevel.WARN;
```
[the remark this answers is not in the document]
Comment (oliver-krauss) [reply api/src/main/java/science/aist/seshat/DefaultLogger.java:38]: Yes INFO is the usual default http://www.slf4j.org/api/org/slf4j/impl/SimpleLogger.html

[the remark this answers is not in the document]
Comment (pointan) [reply api/src/main/java/science/aist/seshat/DefaultLogger.java]: Good point, changed that.

[the remark this answers is not in the document]
Comment (N520) [reply api/src/main/java/science/aist/seshat/DefaultLogger.java]: neat!

**Review by N520 — APPROVED** [review]

**Comment by pointan** [comment]
Note: Please do not directly rebase it after approving. I already require the fix in another project, and will release it directly, after you both approved, and then I do not need an extra branch and ticket to release the bug fix.

**Review by oliver-krauss — APPROVED** [review]

**Pull request merged**